hieradata/common.yaml

```diff
@@ -68,6 +68,7 @@ deployable_applications: &deployable_applications
   travel-advice-publisher: {}
 
 apt_mirror_hostname: 'apt.publishing.service.gov.uk'
+apt_mirror_fingerprint: '3803E444EB0235822AA36A66EC5FE1A937E3ACBB'
```

Comment:
I was wondering what the a fingerprint meant in the context of serving Debian packages, then I realised it's the fingerprint of the GPG key used to sign the packages. I'm not saying you should change it, but `apt_mirror_gpg_key_fingerprint` might be clearer.

Reply:
I'll rename it